src/cheri_pte.sail

```diff
@@ -110,7 +111,7 @@ function update_PTE_Bits(p : PTE_Bits, a : AccessType(ext_access_type), ext : ex
                                   Execute()    => false,
                                   Read(_)      => false,
                                   Write(_)     => true,
-                                  ReadWrite(_) => true
+                                  ReadWrite(_,_) => true
```

Comment:
Annoying, but please update the lines above to have an extra 2 spaces (well, 3 once you add the missing space after the comma on this line).